### DISCUSSION TITLE
[v4] Dark mode?

### DISCUSSION THREAD

**Discussion by ev3rlost** [tailwindlabs/tailwindcss#14615] [Help] [answered]
How to apply [this PR](https://github.com/tailwindlabs/tailwindcss/pull/14567) in the new alpha to make use of dark mode?

**Comment by ParasSolanki** [comment, marked as answer]
In v4 for the dark mode with `class="dark"` selector you have to add following `@variant` css in your global tailwindcss file.

```css
@variant dark (&:where(.dark, .dark *));
``` 

The cool thing is we can customize this easily and if we want dark mode on attribute for example `[data-theme="dark"]`

```css
@variant dark (&:where([data-theme="dark"], [data-theme="dark"] *));
``` 

Here's video where Adam explains this https://www.youtube.com/live/y4i_CXXWLoQ?si=XOexSQT1trFSQxZi&t=2028

**Reply by ev3rlost** [reply]
Didn't know the vid existed! Thanks a bunch ParasSolanki!
